src/rust/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "markupsafe-rust"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[profile.release]
+debug = true
+
+[dependencies]
+pyo3 = "0.22.2"
```

@davidism I guess if you want abi3, can just enable the feature:

```suggestion
pyo3 = { version = "0.22.2", features = ["abi3"] }
```